fd_webserver/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "fd_webserver",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": {
```

Comment:
Yes, only one author, everyone else should be listed as `contributors`. I was meant to make you the author and set Ashley and myself as contributors, this was just my pre-filled snippet. I'll change it now :smiley: